src/components/Dropdown/index.js

```diff
@@ -0,0 +1,70 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['select', 'addItem', 'removeItem'];
+
+const html = `
+  <div class="dropdown-container" data-select="dropdown-container">
+    <div class="dropdown-toggle" data-select="dropdown-toggle">
+      <span class="dropdown-selected" data-select="dropdown-selected"></span>
+      <span class="dropdown-icon" data-select="dropdown-icon"><svg fill="currentColor" stroke-width="0" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" style="overflow: visible; color: currentcolor;" height="1em" width="1em"><path d="M233.4 406.6c12.5 12.5 32.8 12.5 45.3 0l192-192c12.5-12.5 12.5-32.8 0-45.3s-32.8-12.5-45.3 0L256 338.7 86.6 169.4c-12.5-12.5-32.8-12.5-45.3 0s-12.5 32.8 0 45.3l192 192z"></path></svg></span>
+    </div>
+    <ul class="dropdown-options" data-select="dropdown-options">
+    </ul>
+  </div>
+`;
+
+export default function Dropdown({ items = [] } = {}) {
+  Component.call(this, { html, events });
+
+  items.map((item) => this.addItem(item));
+
+  this.selected.get('dropdown-toggle').addEventListener('click', () => {
+    this.selected.get('dropdown-options').classList.add('block');
```

Comment:
`block`? Isso seria pra exibir as opções? Se for troca essa classe de `block` pra `show` ou `visible`, eu imagino que o `block` é sobre o display, mas classes não podem ser nomeadas pensando no estilo que elas carregam, mas sim sobre o que elas representam de fato.